editor.planx.uk/package.json

```diff
@@ -148,12 +147,12 @@
     "eslint-plugin-testing-library": "^5.11.1",
     "husky": "^8.0.3",
     "identity-obj-proxy": "^3.0.0",
+    "jest": "^29.7.0",
```

Comment:
This is required as a peer for the following libraries, one of the removed (unused) libs must have been requiring it as a peer.

```sh
├─┬ ts-jest 29.1.3
│ └── ✕ missing peer jest@^29.0.0
├─┬ vite-jest 0.1.4
│ ├── ✕ missing peer jest@^27.0.0
```